hooks/pre/30-install_redis_scl.rb

```diff
@@ -1,4 +1,4 @@
 # Working around https://tickets.puppetlabs.com/browse/PUP-2169
-if el7? && local_redis?
+if (el7? && local_redis?) || devel_scenario?
```

Comment:
```suggestion
if el7? && (local_redis? || devel_scenario?)
```

The other solution is to modify `local_redis?` to look at `devel_scenario?` which might actually be cleaner.

Same comment for the other file.

Reply:
I considered that. That seemed to just put more logic into local than I wanted. I didn't feel there was really any clean scenario here at present.

Reply:
IMHO the SCL part is el7 specific so at least the suggestion I made still makes sense to me.